app/src/main/java/com/codesoom/assignment/Task/application/TaskService.java

```diff
@@ -0,0 +1,86 @@
+package com.codesoom.assignment.Task.application;
+
+import com.codesoom.assignment.Task.TaskNotFoundException;
+import com.codesoom.assignment.Task.domain.Task;
+import com.codesoom.assignment.Task.domain.TaskRepository;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+
+/**
+ * TaskController 의 요청을 수행합니다.
+ */
+@Service
+@Transactional
+public class TaskService {
+
+    private final TaskRepository taskRepository;
+
+    /**
+     * TaskService 생성자에 TaskRepository 를 주입합니다.
+     * @param taskRepository 생성자에 주입할 TaskRepository DAO
+     */
+    public TaskService(TaskRepository taskRepository) {
+        this.taskRepository = taskRepository;
+    }
+
+
+    /**
+     * 모든 Task 를 리턴합니다.
```

Comment:
만약 `Task` 클래스의 이름이 다른 것으로 바뀌게 되면(`ToDo`라던가) 주석 속의 `Task`도 모두 수정해야겠죠? 자신도 모르는 사이에 주석과 클래스 이름의 강 결합을 만들어 버린 거에요. 다음과 같이 수정하면 어떨까요?
```suggestion
     * 할 일 목록을 리턴합니다.
```